test/Microsoft.Framework.Configuration.Test/ConfigurationHelperTest.cs

```diff
@@ -17,29 +16,12 @@ public void ResolveFilePath()
             var testFile = Path.GetTempFileName();
             var testDir = Path.GetDirectoryName(testFile);
             var testFileName = Path.GetFileName(testFile);
-            var mockSourceRoot = new MockConfigurationBuilder { BasePath = testDir };
-
-            var actualPath = ConfigurationHelper.ResolveConfigurationFilePath(mockSourceRoot, testFileName);
-
-            Assert.Equal(testFile, actualPath);
-        }
-
-        [Fact]
-        public void ThrowWhenBasePathIsNull()
-        {
-            var testFile = "config.j";
             var mockSourceRoot = new MockConfigurationBuilder();
-            var expectErrorMessage = Resources.FormatError_MissingBasePath(
-                testFile,
-                typeof(IConfigurationBuilder).Name,
-                nameof(mockSourceRoot.BasePath));
 
-            var exception = Assert.Throws<InvalidOperationException>(() =>
-            {
-                ConfigurationHelper.ResolveConfigurationFilePath(mockSourceRoot, testFile);
-            });
+            mockSourceRoot.SetBasePath(testDir);
```

Comment:
Same consideration for this test. Either it belong in `Microsoft.Framework.Configuration.FileExtensions.Test` or in `Microsoft.Framework.Configuration.Test` but in a class that is for tests for FileConfigurationExtensions class.